detox/src/android/AndroidExpect.test.js

```diff
@@ -237,6 +237,17 @@ describe('AndroidExpect', () => {
         await expectToThrow(() => e.element(e.by.id('ScrollView161')).scrollTo('noDirection'));
       });
 
+      it('should setDatePickerDate', async () => {
+        await e.element(e.by.type('android.widget.DatePicker')).setDatePickerDate('2019-02-06T05:10:00-08:00', 'ISO8601');
+      });
+
+      it('should not setDatePickerDate given bad args', async () => {
+        await expectToThrow(() => e.element(e.by.type('android.widget.DatePicker')).setDatePickerDate('2019-02-06T05:10:00-08:00', 'ISO8601'));
```

Comment:
Sorry I can't see what is invalid here comparing to the previous happy path... 🤔

Reply:
Me neither. Looks like a copy and paste error. I remove this line.

I wonder if it makes sense to adjust the `expectToThrow ` in this case. Currently it optionally allows the callback to throw an error. It does not require it to do so. If it would, it would have complained in this case.

Does it make sense to adjust `expectToThrow ` to require an error to be thrown in a seperate pr?

Reply:
Thanks, I created #3791 draft for that.